plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestDynamicTable.java

```diff
@@ -183,7 +183,9 @@ public void testFilterWithPushdownConstraint()
     public void testFilterWithUdf()
```

Comment:
Can you please fix pinot tests to use capital letters for SQL keywords like: SELECT, FROM, WHERE? Let's do it as separate Pr.

Reply:
Sure! Will do.

Reply:
Created #17109